libs/wire-subsystems/src/Wire/UserSubsystem.hs

```diff
@@ -71,6 +90,7 @@ data UserSubsystem m a where
   CheckHandles :: [Handle] -> Word -> UserSubsystem m [Handle]
   -- | parses a handle, this may fail so it's effectful
   UpdateHandle :: Local UserId -> Maybe ConnId -> UpdateOriginType -> Text {- use Handle here? -} -> UserSubsystem m ()
+  -- TODO(mangoiv): this can probably go in favour of 'GetAccountsBy'
```

Comment:
have we looked at whether this can go? If we want to postpone it, maybe it should be a `FUTUREWORK`? 
```suggestion
  -- FUTUREWORK(mangoiv): this can probably go in favour of 'GetAccountsBy'
```